source/controllers/representing-multiple-models.md

```diff
@@ -50,3 +52,39 @@ Now we can use this property in our template:
 
 {{longSongCount}} songs over 30 seconds.
 ```
+
+### Representing Several Models
+
+Multiple models can be returned through `Ember.RSVP.hash`. To continue with the above example, if we also wanted to return the `models/album.js` records then our `model` hook would look like this:
+
+```app/routes/songs.js
+export default Ember.Route.extend({
+  model() {
+    return Ember.RSVP.hash({
+      songs: this.store.findAll('song'),
+      albums: this.store.findAll('album')
+    });
+  }
+});
+```
+
+In the `songs` template, we can specify both models and use the `{{#each}}` helper to display
```

Comment:
You could also add the `longSongCount` computed property from the previous example, to make a parallel about how you have to use `model.songs.@each` instead of `model.@each`.